src/app/pages/server-compare/server-compare.component.scss

```
@@ -208,3 +208,21 @@
 .align-baseline {
   vertical-align: baseline;
 }
+
+.compare-dropdown-menu {
+  z-index: 50;
+}
+
+:host ::ng-deep .compare-table-header-cell {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)